yarn.lock

```diff
@@ -9842,6 +9860,13 @@ vue-meta@^2.4.0:
   dependencies:
     deepmerge "^4.2.2"
 
+vue-router-sitemap@0.0.4:
```

Comment:
is this being used?
curious why  `packages.json` was not updated as well in this PR.
was it?

Reply:
Oh no it’s not being used. Thought i got rid of that

Reply:
ok rebuild lock file